src/resources/i18n/en.json

```diff
@@ -416,7 +416,14 @@
     "classes": {
       "major": "Major",
       "minor": "Minor"
-    }
+    },
+    "addOperation": "Add Operation",
+    "editOperation": "Edit Operation",
```

Comment:
A little bit upper in the same file (line 383), there is already something similar:
```
"editoperation": "Edit Operation",
```

please check for duplicates when adding new translation keys, because afterwards it's very difficult to merge them.